tests/e2e/parallel/main_test.go

```diff
@@ -2795,3 +2797,174 @@ func TestScheduledSuiteTimeoutFail(t *testing.T) {
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if *job.Spec.Suspend == true {
+		t.Fatalf("CronJob %s is suspended when it should not be", jobName)
+	}
+
+	// Suspend the `ScanSetting` using the `suspend` attribute
+	scanSetting.Suspend = true
+	if err := f.Client.Update(context.TODO(), &scanSetting); err != nil {
+		t.Fatal(err)
+	}
+
+	// Assert the `CronJob` associated with the `ComplianceSuite` is set to `suspend=true`
+	job = &batchv1.CronJob{}
+	jobName = compsuitectrl.GetRerunnerName(suite.Name)
+	err = f.Client.Get(context.TODO(), types.NamespacedName{Name: jobName, Namespace: f.OperatorNamespace}, job)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if *job.Spec.Suspend == false {
+		t.Fatalf("Expected CronJob %s to be suspended", jobName)
+	}
+
+	// Resume the `ComplianceScan` by updating the `ScanSetting.suspend` attribute to `False`
+	scanSetting.Suspend = false
+	if err := f.Client.Update(context.TODO(), &scanSetting); err != nil {
+		t.Fatal(err)
+	}
+
+	// Assert the `CronJob` associated with the `ComplianceScan` is set to `suspend=false`
+	job = &batchv1.CronJob{}
+	jobName = compsuitectrl.GetRerunnerName(suite.Name)
+	err = f.Client.Get(context.TODO(), types.NamespacedName{Name: jobName, Namespace: f.OperatorNamespace}, job)
+	if err != nil {
+		t.Fatal(err)
+	}
+	if *job.Spec.Suspend == true {
+		t.Fatalf("CronJob %s is suspended when it should not be", jobName)
+	}
+
+}
+
+func TestSuspendScanSettingDoesNotCreateScanAfterBinding(t *testing.T) {
+	t.Parallel()
+	f := framework.Global
+
+	// Creates a new `ScanSetting` with `suspend` set to `True`
+	scanSettingName := framework.GetObjNameFromTest(t) + "-scansetting"
+	scanSetting := compv1alpha1.ScanSetting{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      scanSettingName,
+			Namespace: f.OperatorNamespace,
+		},
+		ComplianceSuiteSettings: compv1alpha1.ComplianceSuiteSettings{
+			AutoApplyRemediations: false,
+			Suspend:               true,
+		},
+		Roles: []string{"master", "worker"},
+	}
+	if err := f.Client.Create(context.TODO(), &scanSetting, nil); err != nil {
+		t.Fatal(err)
+	}
+	defer f.Client.Delete(context.TODO(), &scanSetting)
+
+	// Bind the new `ScanSetting` to a `Profile`
+	bindingName := framework.GetObjNameFromTest(t) + "-binding"
+	scanSettingBinding := compv1alpha1.ScanSettingBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      bindingName,
+			Namespace: f.OperatorNamespace,
+		},
+		Profiles: []compv1alpha1.NamedObjectReference{
+			{
+				Name:     "ocp4-cis",
+				Kind:     "Profile",
+				APIGroup: "compliance.openshift.io/v1alpha1",
+			},
+		},
+		SettingsRef: &compv1alpha1.NamedObjectReference{
+			Name:     scanSetting.Name,
+			Kind:     "ScanSetting",
+			APIGroup: "compliance.openshift.io/v1alpha1",
+		},
+	}
+	if err := f.Client.Create(context.TODO(), &scanSettingBinding, nil); err != nil {
+		t.Fatal(err)
+	}
+	defer f.Client.Delete(context.TODO(), &scanSettingBinding)
+
+	// Assert that no scan takes place
```

Comment:
This assertion and the corresponding implementation in the compliance suite controller needs to be implemented.